panther_analysis_tool/main.py

```diff
@@ -611,6 +619,13 @@ def run_tests(analysis: Dict[str, Any], analysis_funcs: Dict[str, Any],
                 print('\t\t[{}] [{}] {}'.format(
                     test_result[func], func, analysis_funcs[func](test_case)))
 
+    if minimum_tests > 1 and not (
+        [x for x in analysis['Tests'] if x['ExpectedResult']] and
+        [x for x in analysis['Tests'] if not x['ExpectedResult']]):
+        failed_tests[analysis.get('PolicyID') or analysis['RuleID']].append(
+            'Insufficient test coverage: expected at least one passing and one failing test.'
```

Comment:
Phrasing could maybe be a bit clear here; its not looking for a failing test persay but a test that is expecting a failing outcome. Subtle difference so I ignored it here in the interest of brevity, but if anyone has a suggestion for  how to make it more clear without sacrificing brevity I'm all ears.

Reply:
maybe say 'tests that would generate or not generate an alert' or 'positive/negative tests'?

I think we could also remove the period at the end of the sentence